### ISSUE TITLE
How to config HEADLESS mode?

### ISSUE THREAD

**Issue by hongda3141** [ChainSafe/dappeteer#234]
**Describe the problem**
Hi, i tried to config headless mode with code like this,but it doesn't work, the pages come out.How to config `--headless` mode?
```javascript
import puppeteer from 'puppeteer';
import dappeteer from '@chainsafe/dappeteer';

async function main() {
  const DAPPETEER_DEFAULT_CONFIG = {
    metamaskVersion: "10.23.0",
    headless: true,
    args: [
        "--headless=chrome",
    ],
    }
  const browser = await dappeteer.launch(puppeteer, DAPPETEER_DEFAULT_CONFIG);
}

```

Thanks in advance

**System:**
- OS: [ubuntu20.04]
- OS version [e.g. 22]
- NodeJs version [v16]
- dAppeteer version [4.0.1]
- testing framework [e.g jest]
- testing framework version [e.g 26.0]

**Comment by mpetrunic** [comment]
You cannot install extensions on headless mode so headless mode is disabled^^

**Comment by hongda3141** [comment]
> You cannot install extensions on headless mode so headless mode is disabled^^
@mpetrunic Thanks a lot for helping!
By the way, is this a new feature? 
Because I tried to use the `headless` mode in the v2.4.1 and succeeded. 
So is that "headless" mode is not available in the new version of v3.0.0 or v4.0.1?

**Comment by hongda3141** [comment]
that's worked for me by using ` "--headless=new"`:
```javascript
import puppeteer from 'puppeteer';
import dappeteer from '@chainsafe/dappeteer';

async function main() {
  const DAPPETEER_DEFAULT_CONFIG = {
    metamaskVersion: "10.23.0",
    headless: true,
    args: [
        "--headless=new",
    ],
    }
  const browser = await dappeteer.launch(puppeteer, DAPPETEER_DEFAULT_CONFIG);
}

```
